hlx_statics/scripts/lib-adobeio.js

```diff
@@ -193,36 +194,33 @@ export function decorateInlineCodes(element) {
  * Builds all code blocks inside a container
  * @param {*} container The container to inspect
  */
-export function buildCodes(container) {
-  const codes = [...container.querySelectorAll('main > div pre')];
-
+ export function buildCodes(container) {
+  const codes = [...container.querySelectorAll('main > div pre > code')];
   codes.forEach((code) => {
-
-    const parentDiv = code.closest('div');
-    parentDiv.classList.add('code-container');
     const block = buildBlock('code', code.outerHTML);
-
-    if (code) {
-      const wrapperDiv = document.createElement('div');
-      const blockDiv = document.createElement('div');
-
-      wrapperDiv.style.margin = "1em 0";
+    block.classList.add('block');
+    const parentContainer = code.parentElement.parentElement;
+    const pre = parentContainer.querySelector('pre');
+    pre.replaceWith(block);
+
+    if (pre.tagName === 'PRE') {
+      const code_wrapper = document.createElement('div');
+      code_wrapper.style.margin = "1em 0";
+      code_wrapper.classList.add('code', 'block');
```

Comment:
Line 200 should already add code as class name: https://github.com/AdobeDocs/adp-devsite/blob/7afea298d0d7117c15e74e8dfc0a705ee151880d/hlx_statics/scripts/lib-helix.js#L442 

```suggestion
      code_wrapper.classList.add('block');
```